vms/platformvm/txs/executor/camino_tx_executor.go

```diff
@@ -103,7 +103,8 @@ var (
 	errExpiredProposalsMismatch          = errors.New("expired proposals mismatch")
 	errWrongAdminProposal                = errors.New("this type of proposal can't be admin-proposal")
 	errNotPermittedToCreateProposal      = errors.New("don't have permission to create proposal of this type")
-	errInvalidProposal                   = errors.New("proposal is semantically invalid")
+
+	ErrInvalidProposal = errors.New("proposal is semantically invalid")
```

Comment:
is this intended ? I see other errors are not global (no uppercase at the beginning)

Reply:
yes, its other in platformvm package test